src/reducers/books.js

```diff
@@ -0,0 +1,18 @@
+const booksReducer = (state = [], action) => {
+  switch (action.type) {
+    case 'CREATE_BOOK':
+      return [
+        ...state,
+        action.book,
+      ];
+    case 'REMOVE_BOOK':
+      return [
+        ...state.slice(0, state.indexOf(action.book)),
+        ...state.slice(state.indexOf(action.book) + 1),
+      ];
```

Comment:
**OPTIONAL**: Well done! Could [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) help simplify this and make it more expressive (readable)?